components/Media/samples/PipelineVisualFactory.md

```diff
@@ -0,0 +1,15 @@
+---
+title: PipelineVisualFactory
+author: erinwoo  
+description: A helper type that can be used to create sprite visuals with custom Win2D/Composition effects chains and attach them to UI elements. It can create the same visual brushes as the PipelineBrush type, but it can attach them directly on the underlying Visual instance backing a UI element. This can make the XAML code less verbose and more efficient, as there is no need to insert additional elements just so that a brush can be applied to them.
+keywords: pipeline, visual, factory, pipe
+dev_langs:
+  - csharp
+category: Media
+subcategory: Brushes
+discussion-id: 0
+issue-id: 0
+icon: Assets/EffectAnimations.png
+---
+
```

Comment:
Since these get outputted to the docs on learn.microsoft.com too, we should have a bit of background on what this is for and then just have the example.

e.g. https://github.com/CommunityToolkit/Windows/blob/main/components/Primitives/samples/SwitchPresenter.md